Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1725,24 +1739,19 @@ def get_incidents_long_running_execution(client: Client, offenses_per_fetch: int
 
     offenses: list[dict] = []
     if fetch_mode != FetchMode.no_events.value:
-        try:
-            futures = []
-            for offense in raw_offenses:
-                futures.append(EXECUTOR.submit(
-                    enrich_offense_with_events,
-                    client=client,
-                    offense=offense,
-                    fetch_mode=fetch_mode,
-                    events_columns=events_columns,
-                    events_limit=events_limit,
-                ))
-            offenses_with_success = [future.result(timeout=DEFAULT_EVENTS_TIMEOUT * 60) for future in futures]
-            offenses = [offense for offense, _ in offenses_with_success]
-            prepare_context_for_failed_events(offenses_with_success)
-        except concurrent.futures.TimeoutError as e:
-            demisto.error(
-                f"Error while enriching offenses with events: {str(e)} \n {traceback.format_exc()}")
-            update_missing_offenses_from_raw_offenses(raw_offenses, offenses)
+        futures = []
+        for offense in raw_offenses:
+            futures.append(EXECUTOR.submit(
+                enrich_offense_with_events,
+                client=client,
+                offense=offense,
+                fetch_mode=fetch_mode,
+                events_columns=events_columns,
+                events_limit=events_limit,
+            ))
+        offenses_with_success = [future.result() for future in futures]
+        offenses = [offense for offense, _ in offenses_with_success]
+        prepare_context_for_failed_events(offenses_with_success)
```

Comment:
We shouldn't run multiple threads with no `timeout` at all. Besides, setting a timeout is a valid use case for the user - it's ok to set up a timeout along with the retry mechanism, as you might want to put a hard limit on the amount of time you're willing to wait.